frontend/src/components/question-creation/QuestionSidebar.tsx

```diff
@@ -69,24 +76,26 @@ const renderAccordionItem = (items: AccordionItemProps[]) => {
           </Box>
           <AccordionIcon />
         </AccordionButton>
-        {renderAccordianPanel(item.panels)}
+        {renderAccordionPanel(addQuestionElement, item.panels)}
       </AccordionItem>
     );
   });
 };
 
-const CreateQuestionSidebar = ({
-  pageToNavigate,
-}: CreateQuestionSidebarProps): React.ReactElement => {
+const QuestionSidebar = ({
+  addQuestionElement,
+  backPage,
+}: QuestionSidebarProps): React.ReactElement => {
   const history = useHistory();
-  const navigateTo = () => history.push(pageToNavigate);
+  const navigateTo = () => history.push(backPage);
 
   return (
     <VStack
-      minHeight="100vh"
-      boxShadow="8px 4px 4px rgba(193, 186, 186, 0.25)"
+      height="100vh"
```

Comment:
```suggestion
      minHeight="100vh"
```
to fix this issue
<img width="344" alt="image" src="https://user-images.githubusercontent.com/69697180/220232906-74c0f0c3-4ad0-4d8c-a8a2-5047f954b340.png">
<img width="531" alt="image" src="https://user-images.githubusercontent.com/69697180/220232787-5482fc9b-79ec-4338-99c1-221c258011c7.png">